README.md

```diff
@@ -60,7 +60,7 @@ API: [FileInfo](http://www.jfrog.com/confluence/display/RTF/Artifactory+REST+API
   ```
   All this info will be available in the *fileInfoJson* object that is returned as part of the promise resolution.
 
-### uploadFile(repoKey, remotefilePath, localfilePath, forceUpload)
+### uploadFile(repoKey, remotefilePath, fileToUploadPath, forceUpload, checksums)

```

Comment:
If we now have 2 optional param I guess it is better to encapsulate them in an "opts" object.